Base/usr/share/man/man8/pls.md

```diff
@@ -0,0 +1,41 @@
+## Name
+
+pls - Execute a command as root
+
+## Synopsis
+
+```**sh
+$ pls [command]
+```
+
+## Description
+
+Executes a command as the root user (uid and gid 0), given that the user executing `pls` is located in
+the sudoers file.
+
+It is possible to execute commands that contain hyphenated options via the use of `--`, which signifies the
+end of command options. For example:
+
+```sh
+$ pls -- ls -la
+```
+
+## Files
+/etc/sudoers - List of users that can run `pls`
```

Comment:
Why is the file called "sudoers" if we're not using the "sudo" name?

Reply:
It made sense at the time because I couldn't think of a better name, though I agree it would confuse people that now expect `sudo` to work on the system.

Suggestions for a better? "Pleasers" doesn't quite have the same ring to it. Perhaps `plsusers` or just `pls.conf`?